Assets/MRTK/Examples/Demos/UX/Dialog/Scenes/DialogExample.unity

```diff
@@ -156,7 +156,7 @@ GameObject:
   - component: {fileID: 167935246}
   - component: {fileID: 167935248}
   m_Layer: 0
-  m_Name: OpenDialogButtons
+  m_Name: OpenDialogButtonsNear
```

Comment:
The OpenDialogButtonsNear/Far containers look like they still have a RadialView solver attached.  Do these containers need a Radial View attached since it appears the buttons on the slate remain stationary?
![RadialViewDialog](https://user-images.githubusercontent.com/53493796/101944572-c7a22d80-3ba1-11eb-98f7-7be0627352d5.gif)

Reply:
Removed those unused scripts. Thanks!